tests/components/zwave_mqtt/test_init.py

```diff
@@ -0,0 +1,62 @@
+"""Test integration initialization."""
+from homeassistant import config_entries
+from homeassistant.components.zwave_mqtt import DOMAIN, PLATFORMS, const
+
+from .common import setup_zwave
+
+from tests.common import MockConfigEntry
+
+
+async def test_init_entry(hass, generic_data):
+    """Test setting up config entry."""
+    await setup_zwave(hass, fixture=generic_data)
+
+    # Verify integration + platform loaded.
+    assert "zwave_mqtt" in hass.config.components
+    for platform in PLATFORMS:
+        assert platform in hass.config.components, platform
+        assert f"{platform}.{DOMAIN}" in hass.config.components, f"{platform}.{DOMAIN}"
+
+    # Verify services registered
+    assert hass.services.has_service(DOMAIN, const.SERVICE_ADD_NODE)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5460460)